InventoryService/cmd/controllers/item_controller.go

```diff
@@ -0,0 +1,205 @@
+package controllers
+
+import (
+	"encoding/json"
+	"inventoryService/m/v2/cmd/models"
+	"inventoryService/m/v2/cmd/resources"
+	"inventoryService/m/v2/ent"
+	"inventoryService/m/v2/ent/ingridient"
+	"net/http"
+
+	"github.com/gin-gonic/gin"
+)
+
+type ItemController struct {
+	Client *ent.Client
+}
+
+type Response struct {
+	Msg  string      `json:"message"`
+	Data interface{} `json:"data"`
+}
+
+func ResponseJSON(c *gin.Context, httpCode int, msg string, data interface{}) {
+	c.JSON(httpCode, Response{
+		Msg:  msg,
+		Data: data,
+	})
+}
+
+// @BasePath /api/v1
+
+// GetIngridientsList godoc
+// @Summary 	Gets list of all ingridients
+// @Description Get complete list of all ingridients availible for user
+// @Tags 		Inventory manipulation
+// @Accept 		json
+// @Produce 	application/json
+// @Success		200
+// @Router 		/inventory [get]
+func (controller *ItemController) GetIngridientsList(c *gin.Context) {
+	ingridients, err := controller.Client.Ingridient.
+		Query().
+		All(c)
+
+	if err != nil {
+		ResponseJSON(c, http.StatusBadRequest, err.Error(), nil)
+		return
+	}
+
+	ResponseJSON(c, http.StatusOK, "", ingridients)
+}
+
+// @BasePath /api/v1
+
+// GetIngridientsList godoc
+// @Summary 	Gets list ingridients by filter
+// @Description Get complete list of all ingridients availible for user
+// @Tags 		Inventory manipulation
+// @Accept 		json
+// @Produce 	application/json
+// @Param 		item query models.ItemModelFilters true "Item to search for"
+// @Success		200
+// @Router 		/ingridient/search [get]
+func (controller *ItemController) GetIngridientByFilter(c *gin.Context) {
+	var filters models.ItemModelFilters
+	filters.AlcoholMax = 1
```

Comment:
Constant? Shouldn't it be read from params? Or maybe I don't understand something here